source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -214,13 +214,16 @@ def __init__(self, cfg: SimulationCfg | None = None):
         super().__init__(
             stage_units_in_meters=1.0,
             physics_dt=self.cfg.dt,
-            rendering_dt=self.cfg.dt * self.cfg.substeps,
+            rendering_dt=self.cfg.dt * self.cfg.render_interval,
```

Comment:
We should have a check that if the interval is non-positive, it errors out?

Reply:
This edge-case is caught nicely in the simulation context, so should be fine